code/datums/components/mechComp_signals.dm

```diff
@@ -78,15 +78,13 @@
 
 	var/defaultSignal = "1"
 
-/datum/component/mechanics_holder/Initialize(can_manually_set_signal = 0)
+/datum/component/mechanics_holder/Initialize()
 	src.connected_outgoing = list()
 	src.connected_incoming = list()
 	src.inputs = list()
 	src.configs = list()
 
 	src.configs.Add(list(DC_ALL))
-	if(can_manually_set_signal)
-		allowManualSingalSetting()
```

Comment:
Never used, could only be used by a derivative of `mechanics_holder` ... very unlikely.